pkg/duckprovisionedserviceable/fake/fakeprovisionedservice.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2020 VMware, Inc.
```

Comment:
It is not 2020 and I think we have not been placing this in kpack. Can we drop it?